src/main/java/org/takes/http/MainRemote.java

```diff
@@ -49,6 +49,12 @@
 public final class MainRemote {
 
     /**
+     * The format of the error message indicating that the file could not be
+     * deleted.
+     * @checkstyle LineLength (2 lines)
+     */
+    private static final String DEL_FILE_ERR_MSG = "The file '%s' could not be deleted";
```

Comment:
@essobedo brrr... check this out: http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html not exactly the case, but the problem is the same - why the text is exactly the same? these two pieces of code are connected somehow?